# References for E. Consortia
## Team 5 2023 (An Escherichia Coli Based Bioreactor Immune System)

Aiba, H., Fujimoto, S. & Ozaki, N. (1982). Molecular cloning and nucleotide sequencing of the gene for E. coli camp receptor protein. Nucleic Acids Research, 10(4), pp. 1345–1361.

Aoki, S.K., Webb, J.S., Braaten, B.A. and Low, D.A. (2009). Contact-dependent growth inhibition causes reversible metabolic downregulation in Escherichia coli. Journal of bacteriology, 191(6), pp.1777-1786.

Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., Datsenko, K.A., Tomita, M., Wanner, B.L. and Mori, H. (2006). Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular Systems Biology, 2(1).

Chi, H., Wang, X., Shao, Y., Qin, Y., Deng, Z., Wang, L. and Chen, S. (2019). Engineering and modification of microbial chassis for systems and synthetic biology. Synthetic and systems biotechnology, 4(1), pp.25-33.

Csicsery, N. and O'laughlin, R. (2013). BENG 221 -Mathematical Methods in Bioengineering A Mathematical Model of a Synthetically Constructed Genetic Toggle Switch. [online] Available at: https://isn.ucsd.edu/courses/beng221/problems/2013/project-1-A%20Mathematical%20Model%20of%20a%20Synthetically%20Constructed%20Genetic%20Toggle%20Switch.pdf.

Cuthbert, B.J., Hayes, C.S. and Goulding, C.W. (2022). Functional and Structural Diversity of Bacterial Contact-Dependent Growth Inhibition Effectors. Frontiers in Molecular Biosciences, 9.

Datsenko, K.A. & Wanner, B.L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proceedings of the National Academy of Sciences, 97(12), 6640–6645.

Doyle, M.T. & Bernstein, H.D. (2019) Bacterial outer membrane proteins assemble via asymmetric interactions with the BamA β-barrel. Nature Communications 2019 10:1. 10 (1), 1–13. doi:10.1038/s41467-019-11230-9.

Goodall, E.C.A., Robinson, A., Johnston, I.G., Jabbari, S., Turner, K.A., Cunningham, A.F., Lund, P.A., Cole, J.A. and Henderson, I.R. (2018). The Essential Genome of Escherichia coli K-12. mBio, 9(1).

Guzman, L.-M., Barondess, J.J. and Beckwith, J. (1992). FtsL, an Essential Cytoplasmic Membrane Protein Involved in Cell Division in Escherichia coli. Journal of Bacteriology, 174(23), pp.7717–7728.

Hallgren, J., Tsirigos, K.D., Damgaard Pedersen, M., Juan, J., Armenteros, A., Marcatili, P., Nielsen, H., Krogh, A. & Winther, O. (2022) DeepTMHMM predicts alpha and beta transmembrane proteins using deep neural networks. bioRxiv. 2022.04.08.487609. doi:10.1101/2022.04.08.487609.
Hayes, C.S., Koskiniemi, S., Ruhe, Z.C., Poole, S.J. and Low, D.A. (2014). Mechanisms and biological roles of contact-dependent growth inhibition systems. Cold Spring Harbor perspectives in medicine, 4(2), p.a010025.

Hossein Morowvat, M., Babaeipour, V., Rajabi-Memari, H., Vahidi, H. & Maghsoudi, N. (2014). Overexpression of Recombinant Human Beta Interferon (rhINF-β) in Periplasmic Space of Escherichia Coli. Iran J Pharm Res, 13(Suppl), pp. 151-160.

Karstentischer, B., von Einem, J., Kaufer, B. and Osterrieder, N. (2006). Two-step Red-mediated recombination for versatile high-efficiency markerless DNA manipulation in Escherichia coli. BioTechniques, 40(2), pp.191–197.

Khanal, O. and Lenhoff, A.M. (2021). Developments and opportunities in continuous biopharmaceutical manufacturing. mAbs, 13(1), p.1903664. doi:https://doi.org/10.1080/19420862.2021.1903664.

Lau, B.T.C., Malkus, P. and Paulsson, J. (2013). New quantitative methods for measuring plasmid loss rates reveal unexpected stability. Plasmid, 70(3), pp.353–361. doi:https://doi.org/10.1016/j.plasmid.2013.07.007.

Michalska, K., Quan Nhan, D., Willett, J.L., Stols, L.M., Eschenfeldt, W.H., Jones, A.M., Nguyen, J.Y., Koskiniemi, S., Low, D.A., Goulding, C.W. and Joachimiak, A., (2018). Functional plasticity of antibacterial EndoU toxins. Molecular microbiology, 109(4), pp.509-527.

Milo, R. (2013). What is the total number of protein molecules per cell volume? A call to rethink some published values. BioEssays, [online] 35(12), pp.1050–1055. doi:https://doi.org/10.1002/bies.201300066.

Mirdita, M., Schütze, K., Moriwaki, Y., Heo, L., Ovchinnikov, S. & Steinegger, M. (2022) ColabFold: making protein folding accessible to all. Nature Methods 2022 19:6. 19 (6), 679–682. doi:10.1038/s41592-022-01488-1.

parts.igem.org. (n.d.). Help:Terminators/Measurement/Caitlin Conboy - parts.igem.org. [online] Available at: https://parts.igem.org/Help:Terminators/Measurement/Caitlin_Conboy 

parts.igem.org. (n.d.). Help:Terminators/Measurement/Chris Anderson - parts.igem.org. [online] Available at: https://parts.igem.org/Help:Terminators/Measurement/Chris_Anderson.

parts.igem.org. (n.d.). Help:Terminators/Measurement/Jason Kelly - parts.igem.org. [online] Available at: https://parts.igem.org/Help:Terminators/Measurement/Jason_Kelly.

Pierce, B.G., Wiehe, K., Hwang, H., Kim, B.H., Vreven, T. & Weng, Z. (2014) ZDOCK server: interactive docking prediction of protein–protein complexes and symmetric multimers. Bioinformatics. 30 (12), 1771–1773. doi:10.1093/BIOINFORMATICS/BTU097.

Reis, A.C. & Salis, H.M. (2020) An automated model test system for systematic development and improvement of gene expression models. ACS Synthetic Biology. 9 (11), 3145–3156. doi:10.1021/ACSSYNBIO.0C00394/ASSET/IMAGES/MEDIUM/SB0C00394_M002.GIF.

Ruhe, Z.C., Wallace, A.B., Low, D.A. & Hayes, C.S. (2013). Receptor polymorphism restricts contact-dependent growth inhibition to members of the same species. mBio, 4(4).

Salis, H.M., Mirsky, E.A. & Voigt, C.A. (2009) Automated design of synthetic ribosome binding sites to control protein expression. Nature Biotechnology 2009 27:10. 27 (10), 946–950. doi:10.1038/nbt.1568.

Schaefer, A.L., Lappala, C.R., Morlen, R.P., Pelletier, D.A., Lu, T.-Y.S., Lankford, P.K., Harwood, C.S. and Greenberg, E.P. (2013). LuxR- and LuxI-Type Quorum-Sensing Circuits Are Prevalent in Members of the Populus deltoides Microbiome. Applied and Environmental Microbiology, 79(18), pp.5745–5752. doi:https://doi.org/10.1128/aem.01417-13.

Tomasello, G., Armenia, I. & Molla, G. (2020) The Protein Imager: a full-featured online molecular viewer interface with server-side HQ-rendering capabilities. Bioinformatics. 36 (9), 2909–2911. doi:10.1093/BIOINFORMATICS/BTAA009.

Tong, Y., Jørgensen, T.S., Whitford, C.M., Weber, T., et al. (2021). A versatile genetic engineering toolkit for E. coli based on CRISPR-prime editing. Nature Communications, 12(1).

Yamazaki, Y., Niki, H. and Kato, J. (2008). Profiling of Escherichia coli Chromosome Database. Microbial Gene Essentiality: Protocols and Bioinformatics, pp.385–389.

Zhang, X.C. & Han, L. (2016) How does a β-barrel integral membrane protein insert into the membrane? Protein & Cell. 7 (7), 471. doi:10.1007/S13238-016-0273-6.
